src/lib/components/account/sendVerificationEmailModal.svelte

```diff
@@ -1,79 +1,189 @@
 <script lang="ts">
-    import { invalidate } from '$app/navigation';
+    import { invalidate, goto } from '$app/navigation';
     import { Modal } from '$lib/components';
     import { Button } from '$lib/elements/forms';
-    import { addNotification } from '$lib/stores/notifications';
     import { sdk } from '$lib/stores/sdk';
     import { user } from '$lib/stores/user';
     import { get } from 'svelte/store';
-    import { page } from '$app/state';
+    import { page } from '$app/stores';
+    import Link from '$lib/elements/link.svelte';
     import { Card, Layout, Typography } from '@appwrite.io/pink-svelte';
     import { Dependencies } from '$lib/constants';
-    import { onMount } from 'svelte';
+    import { onMount, onDestroy } from 'svelte';
+    import { base } from '$app/paths';
+    import { browser } from '$app/environment';
 
-    let { show = $bindable(false) } = $props();
+    let { show = $bindable(false), email } = $props();
     let creating = $state(false);
     let emailSent = $state(false);
+    let resendTimer = $state(0);
+    let timerInterval: ReturnType<typeof setInterval> | null = null;
 
-    let cleanUrl = $derived(page.url.origin + page.url.pathname);
+    async function logout(redirect = true) {
+        try {
+            await sdk.forConsole.account.deleteSession({ sessionId: 'current' });
+            if (redirect) {
+                await invalidate(Dependencies.ACCOUNT);
+                goto(`${base}/login`);
+            }
+        } catch (error) {
+            console.error('Logout error:', error);
+        }
+    }
+
+    const cleanUrl = $derived($page.url.origin + $page.url.pathname);
```

Comment:
why `page` from `import { page } from '$app/stores';` and not from `import { page } from '$app/state';`?